packages/core/src/CommandManager.ts

```diff
@@ -101,7 +101,7 @@ export class CommandManager {
 
   public createCan(startTr?: Transaction): CanCommands {
     const { rawCommands, state } = this
-    const dispatch = false
+    const dispatch = true
```

Comment:
I believe this is wrong.

If dispatch is true, commands will think that they can modify the `transaction` when they should not be able to.

`dispatch` should probably be named `shouldDispatch` where it being `false` means you should not, as a command, actually apply your change to the editor.